### PR TITLE
Improve acorm

### PR DESCRIPTION
**Pull request by fernandohds564**
- Add option to save extra info in LOCO input file.
 - Adapt code according to lnls-sirius/dev-packages#1063: add check for data update, instead of just timestamp update.

### CODE REVIEW

**Review by murilobalves — APPROVED** [review]

**Review by anacso17 — APPROVED** [review]

File: apsuite/commisslib/meas_bpms_signals.py [hunk 1]
```diff
@@ -329,6 +330,24 @@ def filter_switching_cycles(orb, freq_sampling, freq_switching):
...
         return orb - sw_pert.T
 
+    @staticmethod
+    def simulate_data_decimation(orb, downsampling=12*8):
+        """Simulate data decimation by application of moving average filter.
+
+        Args:
+            orb (numpy.ndarray, (Nsamples, Nbpms)): Target matrix.
+            downsampling (int, optional): Size of the decimation filter.
+                Defaults to 12 (from TbT to FAcq).
```
Comment (anacso17) [inline apsuite/commisslib/meas_bpms_signals.py:340]: typo? 12*8

Comment (fernandohds564) [reply apsuite/commisslib/meas_bpms_signals.py:340]: right, fixed. thanks!

File: apsuite/commisslib/emit_exchange/meas_bo_tune_bpm.py [hunk 2]
```diff
@@ -33,26 +32,26 @@ def __init__(self):
...
-        ftmp = '{0:24s} = {1:9.3f}  {2:s}\n'.format
-        dtmp = '{0:24s} = {1:9d}  {2:s}\n'.format
-        stmp = '{0:24s} = {1:9s}  {2:s}\n'.format
-
-        stg = ''
-        stg += stmp('event', self.event, '')
-        stg += stmp('event_mode', self.event_mode, '')
-        stg += ftmp('event_delay', self.event_delay, '[us]')
-        stg += stmp('trigger_source', self.trigger_source, '')
+        ftmp = "{0:24s} = {1:9.3f}  {2:s}\n".format
```
Comment (xresende) [inline apsuite/commisslib/emit_exchange/meas_bo_tune_bpm.py:35]: @fernandohds564 , why are you replacing single quote with double quote characters? you will be choosing a differente standard we adopt in other repos. what is the rationale ?

Comment (fernandohds564) [reply apsuite/commisslib/emit_exchange/meas_bo_tune_bpm.py:35]: @xresende, please check my PR in dev-packages on linter and formatter configurations.

**Pull request merged**